app/javascript/components/upload/CoordinateLabelFileForm.js

```diff
@@ -33,12 +33,14 @@ export default function CoordinateLabelForm({
         </div>
         <TextFormField label="Name" fieldName="name" file={file} updateFile={updateFile}/>
         <div className="form-group">
-          <label htmlFor={`coordCluster-${file._id}`}>Corresponding clusters / spatial data:</label><br/>
-          <Select options={associatedClusterFileOptions}
-            id={`coordCluster-${file._id}`}
-            value={associatedCluster}
-            placeholder="Select one"
-            onChange={val => updateCorrespondingClusters(file, val)}/>
+          <label className="labeled-select">Corresponding clusters / spatial data:
+            <Select options={associatedClusterFileOptions}
+              data-analytics-name="label-corresponding-cluster"
+              id={`coordCluster-${file._id}`}
+              value={associatedCluster}
+              placeholder="Select one"
```

Comment:
Our other menus with this phrase add an ellipsis, so for consistency...

```suggestion
              placeholder="Select one..."
```

(I personally prefer no ellipsis, but I prefer consistency more.)